### DISCUSSION TITLE
Is it compatible with mods like ‘XPMSSE - Nemesis - Papyrus Stack Fix’

### DISCUSSION THREAD

**Discussion by qmwsaa** [Monitor221hz/Pandora-Behaviour-Engine-Plus#288] [Q&A] [answered]
Is it compatible with mods like ‘XPMSSE - Nemesis - Papyrus Stack Fix’ (https://www.nexusmods.com/skyrimspecialedition/mods/60363) please?

**Comment by Monitor221hz** [comment, marked as answer]
Yes

**Reply by qmwsaa** [reply]
Thank you very much for your reply, I mean, improvement mods like these, installed with Pandora also have an effect, right?

After I installed Pandora, are those scripts no longer needed to improve the mod?
